signer/ctclient/ctclient.go

```diff
@@ -0,0 +1,64 @@
+// +build ctclient
+
+package ctclient
+
+import (
+	"bytes"
+	"encoding/asn1"
+	"encoding/binary"
+	"encoding/pem"
+
+	"crypto/x509"
```

Comment:
Consider running `goimports` on the code before committing --- it provides a canonical ordering for packages, among other things.